### DISCUSSION TITLE
Dark Theme

### DISCUSSION THREAD

**Discussion by gabrielmocanu** [google/docsy#672] [Ideas]
Does docsy have the option at this time for a dark theme?

**Comment by Arhell** [comment]
> Does docsy have the option at this time for a dark theme?

Hi no.

**Comment by LisaFC** [comment]
There's an interesting idea here - at the moment we just have one set of colours in the example site, but someone could create a core "dark theme" SCSS. Would also need to implement something if we wanted site readers to be able to switch between two colour themes in a live site....

**Comment by blonewolfs** [comment]
Atanas Yonkov has an [example ](https://yonkov.github.io/post/add-dark-mode-toggle-to-hugo/) of how they did it. I haven't had a chance to try it out yet but, it is a starting point.
